cmd/cli/plugin/airgapped-network/interface.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package util defines functions to pull/push/List images
+package main
+
+// util defines methods to pull/push/List images (similar to imgpkg)
+
+type ImgpkgUtil interface {
+	printErrorAndExit(err error)
```

Comment:
`printErrorAndExit`, `IsValidaTag`, `UnderscoredPlus`, `ReplaceSlash` & `testing` methods should be moved out of the interface as the behavior of these functions need not be dynamic.